app/controllers/inat_imports_controller.rb

```diff
@@ -175,4 +177,16 @@ def fresh_tracker(inat_import)
     InatImportJobTracker.where(inat_import: inat_import.id).destroy_all
     InatImportJobTracker.create(inat_import: inat_import.id)
   end
+
+  public
+
+  def cancel
+    inat_import = InatImport.find(params[:id])
+    inat_import.update(cancel: true)
+    tracker = InatImportJobTracker.where(inat_import: inat_import).
+              order(:created_at).last
+    # flash_warning(:inat_import_tracker_canceled.l)
+    redirect_to(inat_import_path(inat_import,
+                                 params: { tracker_id: tracker.id }))
+  end
```

Comment:
I'm not sure we do need a Turbo response here, since we're not updating the tracker page, we're redirecting to the import show page. I'm curious what's not working.